### ISSUE TITLE
Rename attribution, change semantics to any text that must be rendered

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1287]
Clients currently generate labels for properties other than information in `metadata`, such as for `attribution`, `related`, and so forth.  These labels cannot be internationalized or provided by the content publisher.  In some instances, the default labels generated by clients based on the property name are confusing or legally problematic, such as `attribution`.  

"Attribution" in the museum domain is almost entirely understood as "the artist of this object is believed to be X".  The British Library has legal requirements around "attribution"s, but need the ability to put rights related information into a field the the client knows it MUST render.

Proposal:  Clients MUST NOT render property names as labels when the value is textual. If the content provider wants a label or heading, then they MUST include it in the value, probably as HTML.

e.g.

```
{"attribution": {"en" ["<span><b>Usage Rights:</b> You can use freely, but please cite the British Library if you use this content elsewhere.</span>"]}
```

**Comment by azaroth42** [comment]
From https://github.com/IIIF/api/issues/1213#issuecomment-329533349

**Comment by azaroth42** [comment]
Eds discussion:  the resulting UI of the proposal would be horribly inconsistent. :-1: to the proposal. New proposals sought.

**Comment by mikeapp** [comment]
Discussed on Community Call 12/20, no further suggestions

**Comment by tomcrane** [comment]
```
"viewingHint": "mustRender"
```
?

**Comment by tomcrane** [comment]
Also nasty mixing of semantics of tags provided by publisher and the UI of a viewer

**Comment by azaroth42** [comment]
Or rather: 

```
 "behavior": "no-labels"
```
?

**Comment by azaroth42** [comment]
Proposal on AV call of 2018-02-06:  Change the semantics of `attribution` to along the lines of "a statement that should be prominently displayed" and the value of the property being the same as a metadata pair.

```
{
  "mandatoryStatement" : { "label": { "en": "Credit"}, "value": {"en": "Provided by Example Organization"}}  
}
```

**Comment by zimeon** [comment]
Attempts at alternatives to `mandatoryStatement` from slack discussion:
  * `requiredStatement`
  * `requiredEntry `
  * `requiredMetadata`
  * `requiredNote`
  * `hi-viz`

**Comment by azaroth42** [comment]
Another option was `mustDisplay`

**Comment by azaroth42** [comment]
Eds:  `requiredStatement` is the color of this bikeshed

**Comment by zimeon** [comment]
and note need to use the same `requiredStatement` in the Image API 3.0

**Comment by azaroth42** [comment]
Closed by #1413

**Comment by zimeon** [comment]
Reviwed on 2018-02-28 Technical Community Call, no objections
